apps/webapp/src/components/send/send-form.tsx

```diff
@@ -1,49 +1,64 @@
 import { Button, Switch } from '@penumbra-zone/ui';
 import { useStore } from '../../state';
-import { amountToBig, sendSelector, sendValidationErrors } from '../../state/send';
+import { sendSelector, sendValidationErrors } from '../../state/send';
 import { useToast } from '@penumbra-zone/ui/components/ui/use-toast';
-import { isPenumbraAddr, uint8ArrayToBase64 } from '@penumbra-zone/types';
+import { isPenumbraAddr } from '@penumbra-zone/types';
 import { InputBlock } from '../shared/input-block.tsx';
 import InputToken from '../shared/input-token.tsx';
 import { LoaderFunction, useLoaderData } from 'react-router-dom';
-import { AssetBalance, getBalancesByAccountIndex } from '../../fetchers/balances.ts';
+import { AccountBalance, getBalancesByAccount } from '../../fetchers/balances.ts';
 import { throwIfExtNotInstalled } from '../../fetchers/is-connected.ts';
-import { Amount } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/num/v1alpha1/num_pb';
+import { useMemo } from 'react';
 
-export const AssetBalanceLoader: LoaderFunction = async (): Promise<AssetBalance[]> => {
+export const AssetBalanceLoader: LoaderFunction = async (): Promise<AccountBalance[]> => {
   await throwIfExtNotInstalled();
-  return await getBalancesByAccountIndex();
+
+  const balancesByAccount = await getBalancesByAccount();
+
+  if (balancesByAccount.length) {
+    useStore.setState(state => {
+      state.send.account = balancesByAccount[0]?.address ?? '';
+    });
+  }
+
+  if (balancesByAccount[0]?.balances.length) {
+    useStore.setState(state => {
+      state.send.asset = balancesByAccount[0]?.balances[0];
+    });
+  }
```

Comment:
Can you add a code comment on what this is for? I'm assuming you want to initialize with defaults in form?